src/components/StatusFilter.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useRef } from 'react'
+import FilterListIcon from '@material-ui/icons/FilterList'
+import Popover from '@material-ui/core/Popover'
+import Checkbox from '@material-ui/core/Checkbox'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import IconButton from '@material-ui/core/IconButton'
+
+const StatusFilter = ({ status, setStatus }) => {
+  const [open, setOpen] = useState(false)
+  const buttonEl = useRef(null)
+
+  const handleStatusChange = (event) => {
+    setStatus({ ...status, [event.target.name]: event.target.checked })
+  }
+
+  return (
+    <>
+      <IconButton ref={buttonEl} onClick={() => setOpen(true)}>
```

Comment:
what is ref={buttonEl}

Reply:
A React hook that can store elements as references. In the Popover, we use this reference to anchor it to the button.